services/src/pro/datasets/external/sentinel_s2_l2a_cogs.rs

```diff
@@ -427,7 +454,7 @@ impl SentinelS2L2aCogsMetaData {
             let end = if i < num_features - 1 {
                 start_times[i + 1]
             } else {
-                start + 1000 // TODO: determine correct validity for last tile
+                start + 1000 // there was no tile before and there is no tile following. Use 1 sec to be safe. We could assume that there is no other tile in the query and use the query length?
```

Comment:
could we use the buffer here also? Or add another buffer to the config?

Reply:
[fea3921](https://github.com/geo-engine/geoengine/pull/968/commits/fea392130e5c99e71f6ce2a3f1292d6d402a4f89)